docs/network/evm-nodes.md

```diff
@@ -7,38 +7,65 @@ The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink
 ## Prerequisites
 
 - Make sure you understand the interaction between different nodes as described in [Etherlink architecture](/network/architecture).
-- Run an Etherlink Smart Rollup node as described in [Running an Etherlink Smart Rollup node](/network/smart-rollup-nodes).
+- If you want to remove the sequencer operator from your Trusted Computing Base (TCB), run an Etherlink Smart Rollup node as described in [Running an Etherlink Smart Rollup node](/network/smart-rollup-nodes).
```

Comment:
```suggestion
- If you want to verify the blueprints that come from the sequencer, run an Etherlink Smart Rollup node as described in [Running an Etherlink Smart Rollup node](/network/smart-rollup-nodes).
Otherwise, the EVM node trusts how the sequencer processes transactions and updates the Etherlink state.
```
Not sure if this is exactly accurate but we can be more specific than the general term TCB.

Reply:
:+1: